lib/ProjectStories.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace biglotteryfund\utils;
+
+use biglotteryfund\utils\ContentHelpers;
+use craft\elements\Entry;
+use League\Fractal\TransformerAbstract;
+
+class ProjectStoriesTransformer extends TransformerAbstract
+{
+    public function __construct($locale)
+    {
+        $this->locale = $locale;
+    }
+
+    private static function getTrailPhoto($entry)
```

Comment:
There's probably a generic function that could be created here once we've standardised on the new hero field + `trailPhoto` pattern. The main thing that changes is the actual crop parameters.